### ISSUE TITLE
FAC, ELAC, SEC buttons not working

### ISSUE THREAD

**Issue by 03vmate** [flybywiresim/a32nx#4418]
**Mod Version**
Latest custom-autopilot from the installer

**Describe the bug**
FAC, ELAC, SEC buttons not working

**Additional context**
Works in the development build, only the autopilot branch is broken. Couldn't find anything saying it's an unimplemented feature in custom-autopilot, so I'm assuming it's a bug.

Was this working before/when did the issue start occurring?
Hasn't worked for the last few months at least.

Is this a problem in the vanilla unmodded game?
Yes

Discord username (if different from GitHub):
GreenMate#4747

**Comment by Benjozork** [comment]
I'm pretty sure this is intentional as we only support normal operations as of the first version.

**Comment by Benjozork** [comment]
@aguther could confirm

**Comment by lukecologne** [comment]
The different laws are currently not supported, and also High speed and high AOA protections are also not implemented.

**Comment by aguther** [comment]
Yes they are currently dummies because the reconfiguration is not supported yet.

**Comment by aguther** [comment]
I don't know how we should treat it for the moment, because it's not really a "bug" in that sense, it's just not supported yet.

**Comment by AirbusA320Pro** [comment]
@aguther maybe make the korrys display INOP as a label, which kinda implies they're not implemented.

**Comment by MisterChocker** [comment]
> @aguther maybe make the korrys display INOP as a label, which kinda implies they're not implemented.

It sounds intuitive but I don't really like that idea.
We clearly state that these laws are not yet implemented so I don't see the need to change buttons...

**Comment by aguther** [comment]
I agree, better they are just dead.

**Comment by aguther** [comment]
Sorry, wanted to press "Comment" not close.

**Comment by Benjozork** [comment]
Do they have a inop. Tooltip

**Comment by Benjozork** [comment]
If not I can add one

**Comment by lukecologne** [comment]
Implemented with #6913
